Cargo.toml

```diff
@@ -25,6 +25,7 @@ phidget-sys = { version = "0.1", path = "phidget-sys" }
 anyhow = { version = "1.0", optional = true }
 clap = { version = "3.2", optional = true }
 ctrlc = { version = "3.2", features = [ "termination" ], optional = true }
+log = "0.4.21"
```

Comment:
You added a dependency, but I don't see any logging in the code you added. If not, please remove this.